nuclear/tools/module.py

```diff
@@ -173,3 +183,74 @@ def generate_test(parts):
     return template.format()
 
 
+def module_unused(**kwargs):
+    rolesPath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'roles'))
+    modulesPath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'module'))
+    basePath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'module')) + os.sep
```

Comment:
yep, that is the last thing I got to do. .`/b` wasn't working on windows, so I pushed so I could work on my laptop later.